README.md

```diff
@@ -1,5 +1,7 @@
 # Pyth Client
 
+**This crate has deprecated. Please use [pyth-sdk-solana](https://github.com/pyth-network/pyth-sdk-rs/tree/main/pyth-sdk-solana) instead.**
```

Comment:
I suggest mentioning that the other crate is ~identical, so it's very easy to migrate. If it's not trivial to migrate, we can add some text about what changed.

Reply:
How does it look like now?